.github/workflows/swift_package_test.yml

```diff
@@ -110,3 +122,27 @@ jobs:
       - name: Build / Test
         timeout-minutes: 60
         run: docker run -v ${{ github.workspace }}:C:\source swiftlang/swift:${{ matrix.swift_version }}-windowsservercore-1809 cmd /s /c  "swift --version & swift test --version & cd C:\source\ & ${{ inputs.windows_build_command }} ${{ inputs.swift_nightly_flags }}"
+
+  musl-build:
+    name: MUSL
+    if: ${{ inputs.enable_musl_checks }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    container: swift:6.0-noble
```

Comment:
The version here must be exactly the same as the SDK below.  As of today (30/10) `swift:6.0-noble` vends Swift 6.0.2 which is not working with the SDK 6.0.1.

I suggest to pass both the Swift version number and the checksum as parameters